.gitignore

```diff
@@ -31,3 +34,6 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+
+# mypy
+mypy-cache
```

Comment:
Git complains here about a missing newline. Not sure what to do about it though.